pylintrc

```diff
@@ -326,7 +326,7 @@ max-attributes=7
 min-public-methods=2
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=22
+max-public-methods=25
```

Comment:
@mtyaka That's cheating : ) When we need to do that, that usually means a class has too much logic on it, and needs to be split. I haven't checked if that's going to solve this for you, but note that there has been some extensive refactoring of the instance class in https://github.com/open-craft/opencraft/pull/63 .